rbc/tests/test_omnisci_math.py

```diff
@@ -53,7 +53,7 @@ def omnisci():
     ('copysign', 'double(double, double)'),
     ('fabs', 'double(double)'),
     ('factorial', 'int64(int64)'),
-    ('floor', 'int64(int64)'),
+    ('floor', 'int64(double)'),
```

Comment:
I guess that's why this test was failing. The correct signature should be `double(double)`. cc @pearu